plugins/gatsby-plugin-generate-doc-json/utils/handlers.js

```diff
@@ -13,23 +20,37 @@ const getAllAttributes = (node) =>
       }, {})
     : {};
 
+const getSrcUrl = (fileRelativePath, url) =>
+  path.join(
+    path.dirname(fileRelativePath.replace('src/content', '')),
+    url.replace('./', '')
```

Comment:
```suggestion
    url
```

You can simplify this to simply `url`. Setting `url` here, even if it has the `./` still works correctly. Try the following in a node console:

```js
path.join('src/content', './images/one.png') // => 'src/content/images/one.png'
path.join('src/content', 'images/one.png') // => 'src/content/images/one.png'
```